netbsd/README.md

```diff
@@ -0,0 +1,34 @@
+NetBSD support in htop(1)
+===
+
+This implementation makes NetBSD use htop(1) without the need of mount_procfs(8).
+
+The implementation has been copied over and modified from the OpenBSD
+implementation in htop(1).
+
+Current implementation mimics the procfs based implementation in stats
+collection.
+
+Make NetBSD no longer masquerade as Linux.
+
+What works
+---
+
+* Builds in NetBSD without any additional patch
+* Uses sysctl(3) and kvm(3) to get basic information
+* Shows basic meters CPU / Memory / Swap
+* Shows basic process listing
+
+What needs to be tested
+---
+
+* Memory being split into used/buffers/cache
+* Basic features of htop(1) like kill / nice / sort / search works as expected
+* If the meter displays are working as expected
+* If the process states are displayed correctly
+* If Thread information and count are displayed correctly
+
+TODO
+---
+
+* Get the implementation reviewed.
```

Comment:
I think at least this section can be dropped.

For the other parts of this file I think those are better of in the PR discussion so they are more visible. If necessary additional issues for specific tasks should be created for those instead.

Reply:
Should be fixed in adf13a2